### PR TITLE
cve fixes

### PR DESCRIPTION
**Pull request by btensay**
CVE fix added.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -186,7 +186,7 @@ dependencies {
   implementation group: 'org.apache.santuario', name: 'xmlsec', version: '3.0.1'
   implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: tomcatEmbedVersion
   implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: tomcatEmbedVersion
-  implementation group: 'org.apache.xmlgraphics', name: 'batik-all', version: '1.15', ext: 'pom'
+  implementation group: 'org.apache.xmlgraphics', name: 'batik-all', version: '1.16', ext: 'pom'spring
```
Comment (hsjhita) [inline build.gradle:189]: Spring at the end accidental?

Comment (btensay) [reply build.gradle:189]: yeah. misplaced search term.

Comment (btensay) [reply build.gradle:189]: fix it

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -42,6 +42,20 @@
     <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-crypto@.*$</packageUrl>
     <vulnerabilityName>CVE-2020-5408</vulnerabilityName>
   </suppress>
+  <suppress>
```
Comment (hsjhita) [inline config/owasp/suppressions.xml:45]: implementation group: 'org.springframework.security', name: 'spring-security-crypto', version: '5.7.5'

Comment (btensay) [reply config/owasp/suppressions.xml:45]: That is right the updated version is 5.7.5. I left it at 5.7.3 as it is the version the cve errors flagged up.  Updated.

Comment (btensay) [reply config/owasp/suppressions.xml:45]: I see. I add it in gradle instead.

**Review by hsjhita — APPROVED** [review]

**Review by GauravChawlaMOJ — APPROVED** [review]

**Pull request merged**